### ISSUE TITLE
Logo

### ISSUE THREAD

**Issue by rdiego26** [yanmagale/narigajs#5]
What do you think create a logo ? :D

**Comment by brunopulis** [comment]
@rdiego26 Maybe something like this. 

![stock-vector-human-nose-vector-isolated-147002735](https://user-images.githubusercontent.com/1204692/31961470-97ca6620-b8d9-11e7-8816-0f67074aa654.jpg)

**Comment by rdiego26** [comment]
@yanmagale

**Comment by yanmagale** [comment]
I love this ...

@brunopulis how you think about only the nose and the air symbols down at noisy?

In this case, logo will pass a inspire sensation

@rdiego26 do you like?

**Comment by rdiego26** [comment]
I completely agree that we could have some things at the bottom of our noses, graphic elements that reminded us of air, nature, or something close to it. Makes sense ?

**Comment by brunopulis** [comment]
@rdiego26 Make totally sense! LOL
